### PR TITLE
[AdaptiveUiCodelab] Use NavigationSuiteScaffold

### PR DESCRIPTION
**Pull request by jdkoren**
Note this component doesn't support a navigation drawer. Keeping `NavigationDrawerContent` composable for now in case we intend to implement support for showing the drawer again.

### CODE REVIEW

File: AdaptiveUiCodelab/gradle/libs.versions.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ kotlinxCoroutinesAndroid = "1.8.0"
 lifecycleViewmodelCompose = "2.7.0"
 lifecycleRuntimeKtx = "2.7.0"
 material3Adaptive = "1.0.0-alpha12"
+material3AdaptiveNavSuite = "1.0.0-alpha07"
```
Comment (alexvanyo) [inline AdaptiveUiCodelab/gradle/libs.versions.toml:14]: Nit: we should update these to beta now

**Review by alexvanyo — CHANGES_REQUESTED** [review]
Keeping the navigation drawer content in the underlying project sounds good to me, but I feel like we should remove it from the codelab if we don't use it. Otherwise it just feels like a bug: "why is this composable here if it is never called?"

File: AdaptiveUiCodelab/gradle/libs.versions.toml (hunk 1 above)
Comment (jdkoren) [reply AdaptiveUiCodelab/gradle/libs.versions.toml]: Done

**Comment by jdkoren** [comment]
> Keeping the navigation drawer content in the underlying project sounds good to me, but I feel like we should remove it from the codelab if we don't use it. Otherwise it just feels like a bug: "why is this composable here if it is never called?"

Removed

**Review by alexvanyo — APPROVED** [review]
Removal looks good to me

**Pull request merged**